source/optional_features/moto_payments/index.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: Take payments over the phone ('MOTO' payments)
+weight: 200
+---
+
+# Take payments over the phone (‘MOTO’ payments)
+
+We can set up your GOV.UK Pay account so you can fill in GOV.UK Pay payment pages with your users’ details.
+
+This means you can take payments:
+
+- over the phone
+- by post
+
+These payments are also known as Mail Order Telephone Order (MOTO) payments.
+
+You cannot take MOTO payments if your payment service provider (PSP) is SmartPay.
+
+## Set up your account
+
+[Contact us](/support_contact_and_more_information/#general-feedback) and we’ll:
+
+- review your [Payment Card Industry Data Security Standards (PCI DSS) assessment](https://docs.payments.service.gov.uk/security/#payment-card-industry-pci-compliance) to check you can take payments over the phone and by post
+- enable MOTO payments in your account
+
+## Create a MOTO payment
+
+Make sure you've collected your user's:
+
+- card number
+- card expiry date
+- card security code
+- name from their card
+- email address - if you’ve [enabled collecting your users’ email addresses](/integrate_with_govuk_pay/#collecting-your-users-39-billing-addresses)
+
+Do the following to create and complete the MOTO payment.
+
+1. If your PSP is Worldpay or ePDQ, make sure you're using an API key from your MOTO account in [your services](https://selfservice.payments.service.gov.uk/my-services).
```

Comment:
I think it might be helpful to somehow highlight the difference if you're using Stripe.

Reply:
Just to check - do you mean we should say something about Stripe users needing to use API keys from their 'normal' account?

Reply:
Hmm, I was thinking more in general terms of making services aware that with Stripe, they won't need separate MOTO gateway accounts.
Actually I think this is alright for now. I'm not sure if services are clever enough to know that the implication with Stripe is that they only need one gateway account.